src/Apps/Order/Routes/__tests__/Respond.test.tsx

```diff
@@ -222,16 +233,103 @@ describe("Offer InitialMutation", () => {
         .props()
         .onChange({ currentTarget: { value: "84838" } } as any)
 
+      expect(commitMutationMock).toHaveBeenCalledTimes(0)
+
       component
         .find(Button)
         .last()
         .props()
         .onClick({})
 
-      // TODO: get rid of window.alert
-      expect(window.alert).toHaveBeenCalledWith(
-        `You decided to COUNTER with 84838.`
+      expect(commitMutationMock).toHaveBeenCalledTimes(1)
+
+      expect(commitMutationMock.mock.calls[0][1].variables)
+        .toMatchInlineSnapshot(`
+Object {
```

Comment:
This formatting is weird 😬.

But `inlineSnapshot` is so cool!

Reply:
Yup gotta put up with the weird formatting, but otherwise the DX of `inlineSnapshot` is :100: